app/controllers/api/v1/moves_controller.rb

```diff
@@ -8,7 +8,8 @@ class MovesController < ApiController
       def index
         moves = Moves::Finder.new(filter_params, current_ability, params[:sort] || {}).call
         # Excludes potentially many court hearing documents to reduce the request size. This was requested specifically by the frontend team.
-        paginate moves, include: MoveSerializer::INCLUDED_ATTRIBUTES.dup.except(:court_hearings), fields: MoveSerializer::INCLUDED_FIELDS
+        includes = MoveSerializer::INCLUDED_ATTRIBUTES.dup - %w[court_hearings]
```

Comment:
`MoveSerializer::INCLUDED_ATTRIBUTES` constant is only reference here, so would be cleaner to simply remove `court_hearings` from that constant defined in `MoveSerializer`.

Reply:
Ignore this comment 🤦